src/NHSD.BuyingCatalogue.Ordering.Api/Startup.cs

```diff
@@ -21,23 +21,21 @@ public sealed class Startup
     {
         private readonly IWebHostEnvironment _environment;
 
+        private readonly IConfiguration Configuration;

```

Comment:
As per our coding standards meeting - shouldn't it be `_configuration` ?

Reply:
Yes it should! 👍